<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1045</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="startStopButton">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>50</y>
      <width>75</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchBottomButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>50</y>
      <width>75</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>寻底</string>
    </property>
   </widget>
   <widget class="QPushButton" name="riseButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>50</y>
      <width>75</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>上升</string>
    </property>
   </widget>
   <widget class="QPushButton" name="runButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>50</y>
      <width>75</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>运行</string>
    </property>
   </widget>
   <widget class="QPushButton" name="midButton">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>50</y>
      <width>75</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>中立位</string>
    </property>
   </widget>
   <widget class="QPushButton" name="downButton">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>50</y>
      <width>75</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>落下</string>
    </property>
   </widget>
   <widget class="QLabel" name="platConnectlabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>716</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>平台连接</string>
    </property>
   </widget>
   <widget class="QLabel" name="sendCountlabel">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>715</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>发送个数</string>
    </property>
   </widget>
   <widget class="QLabel" name="revcCountlabel">
    <property name="geometry">
     <rect>
      <x>857</x>
      <y>715</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>接收个数</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="recvCountEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>715</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="sendCountEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>715</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="platConnButton">
    <property name="geometry">
     <rect>
      <x>78</x>
      <y>712</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:5px groove gray;border-radius:15px;
background-color: rgb(85, 85, 0);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="recvStatusEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>413</x>
      <y>716</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="sendCmdEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>716</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="sendCmdLabel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>716</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>发送命令</string>
    </property>
   </widget>
   <widget class="QLabel" name="recvStatusLabel">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>716</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>平台状态</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="functionTabWidget">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>120</y>
      <width>491</width>
      <height>501</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QwtPlot" name="angleQwtPlot">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>123</y>
      <width>481</width>
      <height>241</height>
     </rect>
    </property>
    <property name="autoReplot">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QwtPlot" name="displacementQwtPlot">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>380</y>
      <width>481</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="hengyaorEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>159</x>
      <y>680</y>
      <width>55</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="hengyilabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>630</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>横移</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="hangxiangrEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>229</x>
      <y>680</y>
      <width>55</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="poslabel">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>652</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>理论位置</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="zongqingrEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>89</x>
      <y>680</y>
      <width>55</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="hengyaotEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>159</x>
      <y>650</y>
      <width>55</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="qianchongrEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>368</x>
      <y>680</y>
      <width>55</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="hangxianglabel">
    <property name="geometry">
     <rect>
      <x>241</x>
      <y>630</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>航向</string>
    </property>
   </widget>
   <widget class="QLabel" name="zongqinglabel">
    <property name="geometry">
     <rect>
      <x>103</x>
      <y>630</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>纵倾</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="hangxiangtEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>229</x>
      <y>650</y>
      <width>55</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="shengjiangrEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>435</x>
      <y>680</y>
      <width>55</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="speedlabel">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>682</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>实际位置</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="hengyirEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>298</x>
      <y>680</y>
      <width>55</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="shengjiangtEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>435</x>
      <y>650</y>
      <width>55</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="shengjianglabel">
    <property name="geometry">
     <rect>
      <x>447</x>
      <y>630</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>升降</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="qianchongtEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>368</x>
      <y>650</y>
      <width>55</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="zongqingtEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>89</x>
      <y>650</y>
      <width>55</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="hengyaolabel">
    <property name="geometry">
     <rect>
      <x>171</x>
      <y>630</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>横摇</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="hengyitEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>298</x>
      <y>650</y>
      <width>55</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="qianchonglabel">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>630</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>前冲</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="para2Edit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>678</x>
      <y>650</y>
      <width>55</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="servocodelabel">
    <property name="geometry">
     <rect>
      <x>548</x>
      <y>682</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>伺服代码</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="code1Edit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>608</x>
      <y>680</y>
      <width>55</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="para1Edit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>608</x>
      <y>650</y>
      <width>55</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="code3Edit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>748</x>
      <y>680</y>
      <width>55</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="code4Edit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>817</x>
      <y>680</y>
      <width>55</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="para6Edit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>954</x>
      <y>650</y>
      <width>55</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="code2Edit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>678</x>
      <y>680</y>
      <width>55</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="para4Edit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>817</x>
      <y>650</y>
      <width>55</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="para3Edit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>748</x>
      <y>650</y>
      <width>55</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label1">
    <property name="geometry">
     <rect>
      <x>614</x>
      <y>630</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1号伺服</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="code5Edit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>887</x>
      <y>680</y>
      <width>55</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="para5Edit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>887</x>
      <y>650</y>
      <width>55</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="code6Edit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>954</x>
      <y>680</y>
      <width>55</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="servoparalabel">
    <property name="geometry">
     <rect>
      <x>548</x>
      <y>652</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>伺服参数</string>
    </property>
   </widget>
   <widget class="QLabel" name="label2">
    <property name="geometry">
     <rect>
      <x>685</x>
      <y>633</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>2号伺服</string>
    </property>
   </widget>
   <widget class="QLabel" name="label3">
    <property name="geometry">
     <rect>
      <x>755</x>
      <y>630</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>3号伺服</string>
    </property>
   </widget>
   <widget class="QLabel" name="label4">
    <property name="geometry">
     <rect>
      <x>823</x>
      <y>630</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>4号伺服</string>
    </property>
   </widget>
   <widget class="QLabel" name="label5">
    <property name="geometry">
     <rect>
      <x>894</x>
      <y>630</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>5号伺服</string>
    </property>
   </widget>
   <widget class="QLabel" name="label6">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>630</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>6号伺服</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
